proto/substrait/algebra.proto

```diff
@@ -760,6 +765,53 @@ message NestedLoopJoinRel {
     JOIN_TYPE_RIGHT_ANTI = 8;
     JOIN_TYPE_LEFT_SINGLE = 9;
     JOIN_TYPE_RIGHT_SINGLE = 10;
+    JOIN_TYPE_MARK = 11;
+  }
+
+  substrait.extensions.AdvancedExtension advanced_extension = 10;
+}
+
+message DuplicateEliminatedGetRel{
+   RelCommon common = 1;
+   ReferenceRel input = 2;
+   repeated Expression.FieldReference column_ids = 3;
+}
+
+message DuplicateEliminatedJoinRel {
```

Comment:
We should add documentation about these two relations to site/docs/relations/physical_relations.md.  Important items to cover:

- How the two are used and if there are any implications for the reference relation (presumably the get would go on the right side of the tree underneath the join).
- Any added fields (I don't think there are any)
- A brief description of how the join works and perhaps a link to the paper.